README.md

```diff
@@ -1,79 +1,114 @@
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+4. You can see the differents directories, with the `ls` command. If you want to see all the *.rb* files built for the challenge, go inside of the *lib* directory :
+
+```shell
+$ cd lib
+```
+Or if you want to see all the tests built for all the code, go inside of the *sepc* directory :
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+```shell
+$ cd lib
+```
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+5. You can open the *.rb* file that you wanted to read and change the code in your text editor or using `vim` :
 
+```shell
+$ vim wanted_file.rb
+```
+Or you can just read the contains of it from the command line with `cat` :
 
-In code review we'll be hoping to see:
+```shell
+$ cat wanted_file.rb
+```
+6. Check that the codes are passing the tests by going back to the root *takeaway_challenge* directory and by running the *spec*.<br/>
+You can check all the code files in one time :
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+```shell
+$ cd takeaway_challenge
+$ rspec
+```
+You can check only one code file at a time :
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+```shell
+$ cd takeaway_challenge
+$ rspec spec/file_name_spec.rb
+```
 
-Notes on Test Coverage
-------------------
+7. Check that the code respect the quality of the *Rubocop* guideline, by running `rubocop` from the *takeaway_challenge* directory :
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+```shell
+$ cd takeaway_challenge
+$ rubocop
+```
```

Comment:
Fantastic readme, very instructive and explanatory. Some spelling and grammar errors could use work but this is minor.

Reply:
Ho yes, I'm still doing few mistakes in English :(